Makefile

```diff
@@ -18,7 +18,7 @@ KUBECTL_CLUSTER := $(shell kubectl config current-context 2> /dev/null)
 LOG_VERBOSITY ?= 1
 
 # Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
-ifeq ($(shell go env GOBIN),)
+ifeq ($(shell go env GOBIN 2>/dev/null),)
```

Comment:
I know this is not strictly related to your PR but what do you think about replacing these four lines with:

```sh
GOBIN:=$(or $(shell go env GOBIN), $(shell go env GOPATH)/bin)
```

Reply:
I like this more concise syntax.
However, I want to make it silent if go is not installed because this is what happens when we call `make show-operator-image` outside the ci container to trigger other jobs here:

https://github.com/elastic/cloud-on-k8s/blob/ba1ea9820af67efaa95c77e5ab03191a94d64f15/.ci/pipelines/build.Jenkinsfile#L70-L77